runner/runner.py

```diff
@@ -45,6 +46,19 @@ def create_batch_client(args: object) -> batch.BatchExtensionsClient:
         base_url=args.BatchAccountUrl,
         subscription_id=args.BatchAccountSub)
 
+def create_keyvault_client(args: object):
+    """
+    Create keyvault client namedTuple with url
+    """
```

Comment:
Can you add the correct rtype and return docs to this and since it returns a type you need to add a ->

Reply:
Done